pkg/scheduler/util.go

```diff
@@ -29,7 +29,7 @@ import (
 )
 
 var defaultSchedulerConf = `
-actions: "allocate, backfill"
+actions: "allocate, backfill, enqueue"
```

Comment:
that'll be better if put `enqueue` action before `allocate` action :)

Reply:
agree!